participant/app/root.tsx

```diff
@@ -141,6 +156,11 @@ export default function App() {
         <ScrollRestoration />
         <Scripts />
         <LiveReload />
+        <img
```

Comment:
I think that since you got js tracking to work, we probably want the pixel tracking to be in `<noscript><img ... /></noscript>` tags

Reply:
When you make this change and disable js in your browser, does the pixel tracking work for you?